### PR TITLE
added vindex interface breaking change to summary notes

### PR DESCRIPTION
**Pull request by harshit-gangal**
## Description

As part of this PR https://github.com/vitessio/vitess/pull/10632 the vindex interface methods were modified to add context to the method parameters.
This PR documents those breaking change in the summary release notes.

## Related Issue(s)

- https://github.com/vitessio/vitess/pull/10632

## Checklist

-   [X] Documentation was added or is not required

### CODE REVIEW

File: doc/releasenotes/15_0_0_summary.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+All the vindex interface methods are changed by adding context.Context as the input parameter.
+
+E.g:
+```go
+Map(vcursor VCursor, .... ) .... 
+	To
+Map(ctx context.Context, vcursor VCursor, .... ) ....
+```
+
+This only impact the users who have added their own vindex implementation.
```
Comment (GuptaManan100) [inline doc/releasenotes/15_0_0_summary.md:16]: impact -> impacts

File: doc/releasenotes/15_0_0_summary.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+E.g:
+```go
+Map(vcursor VCursor, .... ) .... 
+	To
+Map(ctx context.Context, vcursor VCursor, .... ) ....
+```
+
+This only impact the users who have added their own vindex implementation. 
+They would be required to change their implementation with these new interface method expectation.
```
Comment (GuptaManan100) [inline doc/releasenotes/15_0_0_summary.md:17]: expectation -> expectations

**Review by GuptaManan100 — APPROVED** [review]
Everything else looks good to me!

**Comment by harshit-gangal** [comment]
> Do we also need changes on the website?

We do not specify interface on website

File: doc/releasenotes/15_0_0_summary.md (hunk 1 above)
Comment (harshit-gangal) [reply doc/releasenotes/15_0_0_summary.md:16]:
```suggestion
This only impacts the users who have added their own vindex implementation. 
```

File: doc/releasenotes/15_0_0_summary.md (hunk 2 above)
Comment (harshit-gangal) [reply doc/releasenotes/15_0_0_summary.md:17]:
```suggestion
They would be required to change their implementation with these new interface method expectations.
```

**Pull request merged**